README.md

```diff
@@ -16,7 +16,9 @@ $ npm install use-fetch-hooks
 import { useFetch } from 'use-fetch-hooks';
 
 export default () => {
-  const { data, error, loading } = useFetch('https://example/api');
+  const { data, error, loading } = useFetch({
```

Comment:
Do we not plan to accept just the default anymore? 

```js
useFetch('https://example/api');
```

Reply:
Yeah, I was actually having trouble typing this properly. It's expressed in `axios` types [here](https://github.com/axios/axios/blob/master/index.d.ts#L125), but I can't figure out how to express an overloaded function signature that matches that.

Reply:
We can revisit this soon.